### DISCUSSION TITLE
23.10.10 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#26] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - Provider(Spotify)에게 제공받은 [엑세스 토큰 관리에 대한 고민](https://github.com/petit-a-petit/MIML/discussions/25) 
- 내일은 어떤 일을 하나요?
    - PR 올리기
    - 코드 리뷰
    - 소셜 로그인 테스트 코드 작성

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
- 내일은 어떤 일을 하나요?
    - 플레이리스트 등록 유효성 추가
    - 플레이리스트 조회 테스트 작성 및 구현
    - 플레이리스트 수정 테스트 작성 및 구현
    - 플레이리스트 삭제 테스트 작성 및 구현

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 간단한 주문 로직 완료
* 내일은 어떤 일을 하나요?
  * 주문 정책 추가
  * 주문 취소 및 부분취소 로직 추가

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - PR 작성

- 내일은 어떤 일을 하나요?
  - @Transactionaleventlistener 사용하기
  - 구현한 기능 리팩토링 진행
